taxclock/settings.py

```diff
@@ -40,3 +40,8 @@
 # Placeholder image when none found on scrape
 IMG_PLACEHOLDER = 'https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/\
                     placeholder.png'
+
+LOG_FILE = os.getenv('LOG_FILE', 'taxclock.log')
+
+# timeout for the url request.
+TIMEOUT = int(os.getenv('TIMEOUT', 30))
```

Comment:
This env variable wouldn't be able to work on Moprh. It has to begin with `MORPH_`.

Also it doesn't follow the structure we've laid out above with the rest of the variables.